src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/NativeRuntimeEventSource.cs

```diff
@@ -11,8 +11,11 @@ namespace System.Diagnostics.Tracing
     [EventSource(Guid = "5E5BB766-BBFC-5662-0548-1D44FAD9BB56", Name = "Microsoft-Windows-DotNETRuntime")]
     internal sealed partial class NativeRuntimeEventSource : EventSource
     {
+        internal const string EventSourceName = "Microsoft-Windows-DotNETRuntime";
+        internal static NativeRuntimeEventSource Log = new NativeRuntimeEventSource();
```

Comment:
Is there a reason to add this singleton instance?

Reply:
This is just moved from how it was generated in current python script.

Reply:
Ah, spotted the script change 😄